active/RoundedCorner/roundedcorner.md

```diff
@@ -0,0 +1,109 @@
+- These UI elements include basic UIs like buttons that users see them on a surface of the screen at all times.
+- The default radius value we use are 2px for these UI.
+![Button](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Button.png)
+
+**Controls being rounded**
+- Button
+- CheckBox
+- ComboBox
+- DropDownButton
+- Slider
+- SplitButton
+- ToggleButton
+- ToggleSplitButton
+- Flipview
+- GridView
+- ListView
+- TreeView
+- ContentDialog
+- AutoSuggestBox
+- PasswordBox
+- RichEditBox
+- TextBox
+- DatePicker
+- CalendarDatePicker
+- Tab control
+
+*Corners of flyout UI elements*
+- These UI elements are those that appear temporarily on UI like a MenuFlyout or Flyout.
+- The default radius value we use here are 4px.
+![flyout](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Flyout.png)
+
+**Controls being rounded**
+- CalendarDatePicker
+- DatePicker
+- TimePicker
+- Flyout
+- TeachingTip
+- ToolTip
+- DropDownButton
+- SplitButton
+- Slider
+- AutoSuggestBox
+- CommandBarFlyout
+- MenuFlyout
+- ComboBox
+- ColorPicker
+- MediaPlayerElement
+- ContentDialog
+- MenuBar
+- ToggleSplitButton
+
+*Rounding of bars*
+- These UI elements are bar like or line like UI (e.g. ProgressBar).
+- The default radius value we use here are 2px except for slider in ColorPicker which will be 6px because the bar is wider.
+![bars](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Bars.png)
+
+**Controls being rounded**
+- NavigationView
+- Pivot
+- ScrollIndicator
+- ProgressBar
+- Slider
+- ColorPicker
+- MediaPlayerElement
+- WebView (may or may not get it from XAML change, checking)
+
+*When not to round*
+- When a multiple buttons are inside a single button that houses them (e.g. SplitButton).
+![SplitButton](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/SplitButton2.png)
+- When a control is housed inside another container (e.g. ScrollBar's bar and buttons that are part of the ScrollBar container which is also part of a ScrollViewer).
+
+![ScrollBar](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/ScrollBar.png)
+- When flyout UI element that is connected to a UI that invokes the flyout on one side.
+![AutoSuggest](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/AutoSuggest.png)
+
+## Note on keyboard focus rect and shadow
+Currently our default design does not require any special work to round the corners of keyboard focus rectangle and shadow. Using higher corner radius value will not break them functionally, however it is good to be aware of this to avoid unwanted visual glitches that you could introduce with a larger value.
```

Comment:
```suggestion
Currently our default design does not require any special work to round the corners of keyboard focus rectangle and shadow. Using higher corner radius value will not break them functionally, however it is good to be aware of this to avoid unwanted visual glitches that you could introduce with a larger value.

Here is an example of larger corner radius make the shadow look undesirable.
![ContentDialogShadow](https://user-images.githubusercontent.com/6964369/60532329-ee241f80-9cb1-11e9-9268-67036d840eff.png)
```